Sources/BraintreeVisaCheckout/BTConfiguration+VisaCheckout.swift

```diff
@@ -0,0 +1,52 @@
+import VisaCheckoutSDK
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// Contains the remote Visa Checkout configuration for the Braintree SDK.
+extension BTConfiguration {
+    
+    // MARK: - Internal Properties
+    
+    /// Returns the environment used to run the Visa Checkout SDK.
+    var visaCheckoutEnvironment: String? {
+        json?["environment"].asString()
+    }
+
+    /// Determines if the Visa Checkout flow is available to be used. This can be used to determine if UI components should be shown or hidden.
+    var isVisaCheckoutEnabled: Bool {
+        json?["visaCheckout"]["apiKey"].isTrue ?? false
+    }
+
+    /// The Visa Checkout API Key associated with this merchant's Visa Checkout configuration.
+    var visaCheckoutAPIKey: String? {
+        json?["visaCheckout"]["apiKey"].asString()
+    }
+
+    /// Returns the Visa Checkout External Client ID configured in the Braintree Control Panel
+    var visaCheckoutExternalClientID: String? {
+        json?["visaCheckout"]["externalClientId"].asString()
+    }
+
+    /// Returns the supported card types for Visa Checkout to accepted card brands.
+    /// - Parameters:
+    ///   - supportedCardTypes: The card types supported by Visa Checkout.
+    func supportedCardTypesToAcceptedCardBrands(_ supportedCardTypes: [String]) -> [String] {
+        let cardTypeMap: [String: String] = [
+            "visa": "VISA",
+            "mastercard": "MASTERCARD",
+            "discover": "DISCOVER",
+            "american express": "AMEX"
+        ]
+        return supportedCardTypes.compactMap { cardTypeMap[$0.lowercased()] }
+    }
+
+    /// The accepted card brands for Visa Checkout.
+    var acceptedCardBrands: [String]? {
+        guard let supportedCardTypes = json?["visaCheckout"]["supportedCardTypes"].asStringArray() else {
+            return ["VISA"]
```

Comment:
We can return an empty array here and not visa since it's not always a visa card used for this flow. That's what android does: https://github.com/braintree/braintree_android/blob/28a1d05bb730e32c31276effb6992bb8e9f7853f/BraintreeCore/src/main/java/com/braintreepayments/api/core/VisaCheckoutConfiguration.kt#L37C13-L37C70